hicommonwealth/commonwealth
# Added support to filter communities by all available tags in the communities list page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8968

## Description of Changes
Added support to filter communities by all available tags in the communities list page

## "How We Fixed It"
N/A

## Test Plan

Note: there was no figma/designs for this filter so I just added some custom designs 

- Visit communities page
- Use the new "Community Tags" filter
- Make sure it works correctly
- Make sure its intuitive (again added custom UI so want to make sure its descriptive)

## Deployment Plan
N/A

## Other Considerations
N/A
